.gitignore

```diff
@@ -5,6 +5,7 @@
 
 # Ignore all files and folders in root
 *
+!/abs.yaml
```

Comment:
why ignore `abs.yaml`?

Reply:
not ignored, 
so that it could be considered by git, 
otherwise it is ignored (because of `*`)